Doc/Manual/Guile.html

```diff
@@ -30,6 +30,7 @@
 <li><a href="#Guile_nn14">Smobs</a>
 <li><a href="#Guile_nn15">Garbage Collection</a>
 </ul>
+<li><a href="#Guile_nn12">Native Guile pointers</a>
```

Comment:
@wsfulton Ping re anchor numbering when inserting a section - how's this usually handled?